src/ovp_pipeline/commands/doctor.py

```
@@ -1035,6 +1035,98 @@ def _promotion_health_payload(vault_dir: Path | None, *, pack_name: str) -> dict
+            decision = evaluate_concept(
+                entry,
+                pack=pack,
+                registry=registry,
+                evidence_kinds=kinds_by_id.get(entry.slug, frozenset()),
+                has_open_contradiction=entry.slug in disputed_ids,
+            )
+            if decision.lane == LANE_ESCALATE:
+                expected_concepts_queue.add(entry.slug)
+            elif decision.lane == LANE_REJECT:
+                expected_rejected_queue.add(entry.slug)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)